vector/src/main/java/im/vector/VectorApp.java

```diff
@@ -193,6 +188,8 @@ public void onCreate() {
             Stetho.initializeWithDefaults(this);
         }
 
+        Realm.init(this);
```

Comment:
Why do we need that here ? It should be hidden, isn't it?

Reply:
yes, but why not? the init() will do think only once. And maybe we will use Realm in Riot for other usage... Do you prefer me to remove this call here?

Reply:
I don't want to see Realm in Riot if it's not used directly yeah...